addons/account/wizard/account_payment_register.py

```diff
@@ -636,9 +638,10 @@ def _get_total_amounts_to_pay(self, batch_result):
             lines |= value['line']
 
         return {
-            'amount_by_default': common + by_default,
-            'amount_for_difference': common + for_difference,
-            'full_amount': common + for_difference + full_amount,
+            'amount_by_default': abs(common + by_default),
+            'amount_for_difference': abs(common + for_difference),
+            'full_amount': abs(common + by_default + full_amount),
+            'full_amount_for_difference': abs(common + for_difference + full_amount),
```

Comment:
no clue... can you add comments/docstring to explain what are those returned value for? :pray: